src/main/deploy/compConfig.txt

```diff
@@ -20,4 +20,14 @@ backRightTurningEncoderPort = 1
 frontLeftOffset = 0.0
 frontRightOffset = 0.7853
 backLeftOffset = -0.0884
-backRightOffset = 2.79
\ No newline at end of file
+backRightOffset = 2.79
+
+FrontShooterFeed_ID = 42
+BackShooterFeed_ID = 43
+
+FrontShooterFeedUp = 0.5
+BackShooterFeedUp = 0.5
+FrontShooterFeedDown = 0.5
+BackShooterFeedDown = 0.5
+FrontShooterFeedStop = 0
```

Comment:
Since this variable doesn't need to be modified depending on the robot, it would be helpful to set this variable where it will be used, instead of the config files to prevent them from becoming cluttered.